pages/functional/adapter/factorial.js

```diff
@@ -0,0 +1,17 @@
+import { countFactorial } from "../js/factorial.js";
+
+document.getElementById("form").addEventListener("submit", function (event) {

```

Comment:
Turn back this file into `functional` folder. I'll explain it later at the end of this review.